py/gpu_specter/test/parity.py

```diff
@@ -0,0 +1,140 @@
+#!/usr/bin/env python
+
+import argparse
+import os
+
+import numpy as np
+import matplotlib as mpl
+mpl.use('Agg')
+import matplotlib.pyplot as plt
+import matplotlib.colors as mcolors
+
+import fitsio
+
+def read_frame(filename):
+    """
+    Simple fits file reader
+
+    TODO: move to gpu_specter.io
```

Comment:
Would you like to address this TODO prior to merging?  Moving `read_frame` to `gpu_specter.io` would be useful for the spex script too.

Reply:
done!